Formula/orientdb.rb

```diff
@@ -39,4 +39,14 @@ def install
   def caveats
     "Use `orientdb <start | stop | status>`, `orientdb-console` and `orientdb-gremlin`."
   end
+
+  test do
+    begin
+      system "#{bin}/orientdb", "start"
```

Comment:
It looks like there's a way to turn off the system file logging using something like `java -Dlog.file.level=OFF` http://orientdb.com/docs/2.2/Logging.html#using-the-jvm

Any tips on how I can run this command from Ruby would be appreciated.